theano/gpuarray/opt.py

```diff
@@ -943,20 +1298,27 @@ def local_inplace_sparseblockouter(node):
 
 
 # This deals with any abstract convs that have a transfer somewhere
-@register_opt('fast_compile')
+@register_opt('fast_compile', 'conv_dnn')
```

Comment:
This should not be tagged conv_dnn, since it does not involve cudnn.
In fact, in the future we want to port GpuCorrMM to the new back-end, wit optimizations replacing AbstractConv2d Ops with CorrMM.